packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -428,17 +455,25 @@ export const rebalanceMinCostFlowSteps = async (
         // XXX: revisit https://github.com/Agoric/agoric-sdk/pull/11953#discussion_r2383034184
         case 'poolToEvm':
         case 'evmToPool': {
+          const operationType = getOperationType(edge.feeMode);
+          const protocol = getProtocolFromPlace(
+            edge.feeMode === 'poolToEvm' ? edge.src : edge.dest,
+          );
           const feeValue = await gasEstimator.getWalletEstimate(
             chainOf(edge.dest) as AxelarChain,
+            operationType,
+            protocol,
           );
```

Comment:
maybe move the logic for getting the protocol to `buildBaseGraph` and define `protocol` as part of the edge, so we dont calculate it here
```suggestion
          const feeValue = await gasEstimator.getWalletEstimate(
            chainOf(edge.dest) as AxelarChain,
            operationType,
            edge.protocol,
          );
```

Reply:
@frazarshad but `edge` is not something that will a `protocol`. Any other proposal?